pkg/models/querybuilder_tag.go

```diff
@@ -33,6 +33,14 @@ func (qb *TagQueryBuilder) Create(newTag Tag, tx *sqlx.Tx) (*Tag, error) {
 	if err := tx.Get(&newTag, `SELECT * FROM tags WHERE id = ? LIMIT 1`, studioID); err != nil {
 		return nil, err
 	}
+	movieID, err := result.LastInsertId()
+	if err != nil {
+		return nil, err
+	}
+
+	if err := tx.Get(&newTag, `SELECT * FROM tags WHERE id = ? LIMIT 1`, movieID); err != nil {
+		return nil, err
+	}
```

Comment:
I deleted the code duplicate but seems that movieID should be change by tagID. But not sure because it is like this. Maybe somebody should review this better than me.

Reply:
Yeah, it's probably best to just leave it like it is and correct it in a different PR.